src/test/test-bpf-token.c

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <bpf/bpf.h>
+#include <fcntl.h>
+
+#include "fd-util.h"
+#include "tests.h"
+
+int main(void) {
+#if __LIBBPF_CURRENT_VERSION_GEQ(1, 5)
+        _cleanup_close_ int bpffs_fd = -EBADF, token_fd = -EBADF;
+
+        ASSERT_OK_ERRNO(bpffs_fd = open("/sys/fs/bpf", O_RDONLY));
+        ASSERT_OK(token_fd = bpf_token_create(bpffs_fd, NULL));
```

Comment:
No need to use these helpers, use DEFINE_MAIN_FUNCTION() and then just return regular errors instead of using the assertion macros, then you don't have to exclude the coredumps either.